test/exceptions/Makefile

```diff
@@ -1,6 +1,11 @@
 include ../common.mak
 
 TESTS:=stderr_msg unittest_assert
+ifeq ($(OS)-$(BUILD),linux-debug)
```

Comment:
Will the ELF backtrace fail w/o debug info? In that case it should fallback to the old .dynamic based symbols.